apache-demo/docs/adopt.md

```diff
@@ -0,0 +1,177 @@
+    |---------------------|--------------------------------------------|
+    | id                  | 5265adec-503e-4e8a-885a-91ba99ee57b0       |
+    | stack_name          | apachedemo                       |
+    | description         | Base infrastructure for an Apache2 example |
+    | creation_time       | 2020-09-18T13:54:59Z                       |
+    | updated_time        | None                                       |
+    | stack_status        | CREATE_IN_PROGRESS                         |
+    | stack_status_reason | Stack CREATE started                       |
+   
+    2. Check that the OpenStack stack is created successfully by entering:
+    ```
+    openstack stack list -c "ID" -c "Stack Name" -c "Stack Status"
+    ```
+3. To communicate with your OpenStack Cluster, on-board it as a deployment location in IBM Telco Network Cloud - Orchestration.
+    1. Log in to your IBM Telco Network Cloud - Orchestration UI.
+    2. From the navigation, select Deployment Locations.
+    3. Click Add.
+    4. Enter a name for the location.
+    5. For the Resource Manager field, select Brent.
+    6. For the Infrastructure field, select OpenStack.
+    7. Add infrastructure properties. These properties are available from your OpenStack's RC file, found under the API section in OpenStack. For example, see these sample values:
+    ```
+    "os_auth_project_name": "test",
+    "os_auth_project_domain_name": "default",
+    "os_auth_username": "admin",
+    "os_auth_password": "password",
+    "os_auth_user_domain_name": "Default",
+    "os_auth_api": "identity/v3",
+    "os_api_url": "http://9.46.89.226",
+    "os_auth_project_id": "701600a059af40e1865a3c494288712a"
+    ```
+    8. Click Save.
+4. Upload the Apache Demo Resource Package. The resource package contains the apache-demo assembly descriptor, the resource descriptor for the Apache2 server, a heat template, and configuration files. For an example resource package, see IBM/tnco-package-examples/Apache-demo.  
+You upload the demo resource package to IBM Telco Network Cloud - Orchestration by using the lmctl command-line tool. For more information about the lmctl command-line tool, see https://github.com/IBM/lmctl).  
+To upload the demo resource package, follow these steps:  
+    1. Unpack the package-examples on the client computer where lmctl is installed and configured.
+    2. Browse to the tnco-package-examples/apache-demo directory.
+    3. Push the assembly to IBM Telco Network Cloud - Orchestration with the following command:
+    ```
+    lmctl project push env
+    ```
+    Where env is the name of your IBM Telco Network Cloud - Orchestration environment that you set up in your lmctl configuration.  
+  
+5. Adopt the stack by way of the REST API. For this procedure, use the curl CLI commands to access the IBM Telco Network Cloud - Orchestration API.
+To access the REST API, retrieve an Access Token for Authorization.
```

Comment:
Sorry to be picky, but should we use REST API here, given the confusion around REST vs RPC components of the API? Perhaps we should settle on something like `TNC-O API`?